src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1706,6 +1706,16 @@ private DetectProperties() {
             .setGroups(DetectGroup.RAPID_SCAN, DetectGroup.BLACKDUCK_SERVER, DetectGroup.BLACKDUCK, DetectGroup.GLOBAL)
             .setCategory(DetectCategory.Advanced)
             .build();
+    
+    public static final NullablePathProperty DETECT_STATUS_JSON_OUTPUT_PATH =
+            NullablePathProperty.newBuilder("detect.status.json.output.path")
+                .setInfo("Status.json Output Path", DetectPropertyFromVersion.VERSION_8_1_0)
+                .setHelp(
```

Comment:
The code that converts the info field apparently doesn't handle "." correctly, so the link from the "all properties" page to the detail (just the anchor... it gets to the right page, but not the right spot on it) isn't working. Not sure how hard it'd be to fix the underlying issue; I'd be comfortable with just avoiding using "." in that field until we do.